tests/testsuite.src/syn_misc.at

```diff
@@ -9075,3 +9075,60 @@ AT_CHECK([$COMPILE_ONLY -fpicture-l=warning prog.cob], [0], [],
 ])
 
 AT_CLEANUP
+
+# " (close unclosed quotation above)
+AT_SETUP([AREACHECK / NOAREACHECK directives])
+AT_KEYWORDS([areacheck])
```

Comment:
this duplicates the keyword that gets in from `AT_SETUP`, but "directive" (= singular) would be useful.